GUI/Main.cs

```diff
@@ -113,7 +113,10 @@ private void ConflictsUpdated(Dictionary<GUIMod, string> prevConflicts)
                         cell.ToolTipText = null;
                     }
                     row.DefaultCellStyle.BackColor = Color.Empty;
-                    ModList.InvalidateRow(row.Index);
+                    if (row.Visible)
+                    {
+                        ModList.InvalidateRow(row.Index);
```

Comment:
Why are we invalidating rows that _were_ conflicts if the next step marks all currently conflicting mods? Won't this leave us with no longer conflicting rows being invalidated?  The comment you previously added above says:

```
// Mark old conflicts as non-conflicted
// (rows that are _still_ conflicted will be marked as such in the next loop)
```

Reply:
"Invalidate" in this case just means "this row needs to be redrawn".
Rows that previously were conflicted and no longer are *should* be invalidated, because previously they had a red background and they now have a white background and should be redrawn.